### ISSUE TITLE
in W3C, there are similar support as HasSessionDetails in JWP

### ISSUE THREAD

**Issue by helenren** [appium/java-client#1689]
## Description

In our test, we use HasSessionDetails to get some information related to the tested device, like device screen size and density for Android. While, when we switch from JSONWP to W3C, noticed that  HasSessionDetails is JWP only. Wondering, is there any similar support in W3C that we can utilize?

## Environment

* Java client build version or git revision if you use some snapshot: 7.6
* Appium server version or git revision if you use some snapshot: 1.17.1
* Desktop OS/version used to run Appium if necessary: mac os
* Mobile platform/version under test: Android
* Real device or emulator/simulator: both

**Comment by KazuCocoa** [comment]
Yes, it is not in W3C for now. A proposal to https://github.com/w3c/webdriver is needed.

As a workaround, you can define the endpoint with https://github.com/appium/java-client/blob/de3ded4d2c149cfa8ccf94d0618afd7b7e79e560/src/main/java/io/appium/java_client/AppiumDriver.java#L200 for now to appium.
